Orange/widgets/data/owrandomize.py

```diff
@@ -0,0 +1,119 @@
+import random
+
+from AnyQt.QtCore import Qt
+from AnyQt.QtWidgets import QSizePolicy
+
+from Orange.data import Table
+from Orange.preprocess import Randomize
+from Orange.widgets.settings import Setting
+from Orange.widgets.widget import OWWidget
+from Orange.widgets import gui
+
+
+class OWRandomize(OWWidget):
+    name = "Randomize"
+    description = "Randomize features, class and/or metas in data table."
+    icon = "icons/Random.svg"
+    priority = 2100
+
+    inputs = [("Data", Table, "set_data")]
+    outputs = [("Data", Table)]
+
+    resizing_enabled = False
+    want_main_area = False
+
+    shuffle_class = Setting(1)
+    shuffle_attrs = Setting(0)
+    shuffle_metas = Setting(0)
+    scope_prop = Setting(80)
+    random_seed = Setting(0)
+    auto_apply = Setting(True)
+
+    def __init__(self):
+        super().__init__()
+        self.data = None
+
+        # GUI
+        box = gui.vBox(self.controlArea, "Shuffle")
+        self.class_check = gui.checkBox(
+            box, self, "shuffle_class", "Classes",
+            callback=self._shuffle_check_changed)
+        self.attrs_check = gui.checkBox(
+            box, self, "shuffle_attrs", "Features",
+            callback=self._shuffle_check_changed)
+        self.metas_check = gui.checkBox(
+            box, self, "shuffle_metas", "Metas",
+            callback=self._shuffle_check_changed)
+        gui.separator(box, 10, 10)
```

Comment:
I'd move this checkbox (and the separator) to the other box. It could be in either of them, but in the second it's the only checkbox, so it's less confusing.